etc/skeleton/driver/impl/impl.go

```diff
@@ -0,0 +1,8 @@
+package impl
+
+import "gopkg.in/bblfsh/sdk.v1/sdk/driver"
+
+func init() {
+	// Can be overridden to link a native driver into a Go driver server.
```

Comment:
I would have left this for another PR since this one is fat enough and this functionality is not directly related with the background of it.

Reply:
This is a pretty small change, and it's required to make native integration tests to work.